README.md

```diff
@@ -282,24 +281,20 @@ W3C submission:
 ```
 
 
-This example RDF states licensing information about Lawrence Lessig's blog in a
-machine-readable way. Lessig's blog (the *subject*) is licensed (the *predicate*)
-under a Creative Commons Attribution 3.0 license (the *object*). ccREL is a schema provided by
-Creative Commons that allows defining context for machines on how to interpret licensing information
-for documents on the web. ccREL itself is implemented using RDF Schema, which is a
-vocabulary for describing properties and classes of RDF resources. By using the [Web
-Ontology Language](https://www.w3.org/TR/owl-features/) (OWL), concepts from other RDF
-schemata can be included in a domain-specific RDF schema, creating a global ontology of semantically
-structured data.
+This example triplet states licensing information about Lawrence Lessig's blog in a machine-readable
+way. Lessig's blog (the *subject*) is licensed (the *predicate*) under a [Creative Commons
+Attribution 3.0](https://creativecommons.org/licenses/by/3.0/) license (the *object*). Provided by
+Creative Commons, ccREL is just one example of a context-providing schema that is built on top of
+RDF. Others, such as the [Web Ontology Language](https://www.w3.org/TR/owl-features/) (OWL), also
```

Comment:
`Others, such as the Web Ontology Language`

I think the comparison between OWL and ccREL is not accurate.
ccREL is specifiying a certain real-life domain as RDF.
OWL is superset of RDF that allows to define an ontology using RDF.

Meaning, they're two different things.